resources/views/components/buttons/select.blade.php

```diff
@@ -0,0 +1,20 @@
+{{-- components.buttons.select
+
+Renders a button like a select dropdown. Most likely used in conjunction with the components.drop-down
+You can customize all the html and css classes but YOU MUST KEEP THE BLADE AND LIVEWIRE DIRECTIVES,
+it is using the variant helper to get the styles for each variant
+it could be primary, primary-light
+
+You can customize the variants classes in config/laravel-views.php
+
+props
+ - title 
+ - block
+ - variant --}}
+<button
```

Comment:
I think it would be even better to be able to add an icon to the main button component, the icon could be passed as a prop, what do you think?